src/Search/Search.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import TextField from '@material-ui/core/TextField';
+import CardList from '../CardsInfo.json';
+import '../Styles/Search.scss';
+
+const Search = ({ setItems }) => {
+  const handleChange = (e) => {
+    const str = e.currentTarget.value;
```

Comment:
What is `str`? Very vague name. Name your variables properly. `arr`, `str` etc are very poor names.